css/default.css

```diff
@@ -1210,7 +1218,8 @@ div#defaultsidebar .mysidebar .sidebarbuttoncontainer {
 }
 
 #jumpto {
-  flex: 1 !important; /* TODO: Disable “important” */
+  flex: 1 !important;
```

Comment:
Please keep the TODO behind the declaration.

(We need to document this, but if a TODO is related to a specific declaration, then it should follow it on the same line, or precede it with a colon—but not be put on the next line, as this looks like it relates to a different declaration altogether. Let’s agree on a convention.)